build-id-versioning/starter/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"github.com/pborman/uuid"
+
+	"go.temporal.io/sdk/client"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Get task queue name from CLI arg
+	taskQueue := os.Args[1]
+	if taskQueue == "" {
+		log.Fatalln("Must provide task queue name as first and only argument")
+	}
+	log.Println("Using task queue", taskQueue)
+
+	// The client is a heavyweight object that should be created once per process.
+	c, err := client.Dial(client.Options{
+		HostPort: client.DefaultHostPort,
+	})
+	if err != nil {
+		log.Fatalln("Unable to create client", err)
+	}
+	defer c.Close()
+
+	// First, let's make the task queue use the build id versioning feature by adding an initial
+	// default version to the queue:
+	err = c.UpdateWorkerBuildIdCompatibility(ctx, &client.UpdateWorkerBuildIdCompatibilityOptions{
+		TaskQueue: taskQueue,
+		Operation: &client.BuildIDOpAddNewIDInNewDefaultSet{
```

Comment:
That is a mouthful, heh. Would have supported something like `BuildIDOpAddSet`, but this was good enough for the SDK review so good enough here. It's not very Go like to usually have these full-sentence identifiers.